app/drugs/[name]/AddFactForm.tsx

```diff
@@ -0,0 +1,41 @@
+import styles from './AddFactForm.module.css';
+import { serverClient as supabase } from '@/app/supabaseClient';
+
+type Props = {
+  drugId: number;
+};
+
+const AddFactForm = ({ drugId }: Props) => {
+  const addFact = async (data: FormData) => {
+    'use server';
+
+    const { error } = await supabase
+      .from('facts')
+      .insert({ description: data.get('description'), drug_id: drugId });
+
+    if (error) throw new Error(error.message);
+  };
+
+  return (
+    <>
+      <form
+        className={styles.form}
+        // @ts-expect-error nextjs server action are not correctly typed
+        action={addFact}
+      >
+        <textarea
+          name="description"
+          className={styles.textarea}
```

Comment:
It's not very semantic to use the html tag to name the classes.

`description` or `description_input` would be more appropriate